src/skspatial/objects/vector.py

```diff
@@ -345,16 +345,16 @@ def angle_between(self, other: array_like) -> float:
         return math.acos(cos_theta)
 
     @np_float
-    def angle_signed(self, other: array_like) -> float:
+    def angle_signed(self, other: array_like, positive_direction: Optional[array_like] = None) -> float:
```

Comment:
I think it would be clearer if you added a separate method called `angle_signed_3d`, which takes the `positive_direction` vector as a positional argument. Then you can leave the original method `angle_signed` as it was.